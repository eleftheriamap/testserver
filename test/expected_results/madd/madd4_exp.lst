
test/exp/test/gen/madd/madd4_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	5158fa94 	sub	w20, w20, #0x63e, lsl #12
   4:	110e31ad 	add	w13, w13, #0x38c
   8:	113f6442 	add	w2, w2, #0xfd9
   c:	9b0d0a8b 	madd	x11, x20, x13, x2
  10:	8a000000 	and	x0, x0, x0
