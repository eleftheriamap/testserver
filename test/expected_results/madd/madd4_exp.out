Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000fd9
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 0000038a9dc18fd9
X12    = 0000000000000000
X13    = 000000000000038c
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 00000000ff9c2000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 0000000000000010
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0x5158fa94
0x00000004 : 0x110e31ad
0x00000008 : 0x113f6442
0x0000000c : 0x9b0d0a8b
0x00000010 : 0x8a000000
